scripts/checks.sh

```diff
@@ -23,6 +23,10 @@ mountpoint="/var/lib/${ENG}"
 
 external_fqdn="0.resinio.pool.ntp.org"
 
+# workaround for self-signed certs, waiting for https://github.com/balena-os/meta-balena/issues/1398
+TMPCRT=/tmp/balenaRootCA.crt
+echo "${BALENA_ROOT_CA}" | base64 -d > "${TMPCRT}"
```

Comment:
Is it possible to use `mktemp` to create this file? Since we force-remove this file up ahead, I'm concerned about accidentally (or deliberately) removing something we shouldn't (symlink attack, two processes running simultaneously, etc).

Reply:
Yeah, that's a much better approach. I kind of forgot we have that on the device :facepalm:, good catch!